src/test/java/org/elasticsearch/deleteByQuery/DeleteByQueryTests.java

```diff
@@ -194,4 +189,14 @@ public void testDeleteByTermQuery() throws ExecutionException, InterruptedExcept
     private static String indexOrAlias() {
         return randomBoolean() ? "test" : "alias";
     }
+
+    private void assertSyncShardInfo(ActionWriteResponse.ShardInfo shardInfo, NumShards numShards) {
+        assertThat(shardInfo.getTotal(), equalTo(numShards.totalNumShards));
+        assertThat(shardInfo.getSuccessful(), greaterThanOrEqualTo(1));
```

Comment:
I don't know why this wasn't using `numShards.numPrimaries`, let me change that!